assets/block/init.php

```diff
@@ -53,7 +53,6 @@ function bctt_block_editor_assets() {
 			),
 			'urlcustom' => array(
 				'type' => 'string',
-				'default' => false
```

Comment:
I added this to get rid of front end notices reported here: https://wordpress.org/support/topic/notice-undefined-variable-26/ 

Please explain why we are removing it now.

Reply:
So far we have been duplicating attributes on client and server side. Now we are using only server-side attribute registration. 

Have tested with debug mode was unable to produce the notices.